ios/RNCallKeep/RNCallKeep.m

```diff
@@ -118,10 +119,11 @@ - (void)sendEventWithNameWrapper:(NSString *)name body:(id)body {
     if (_hasListeners) {
         [self sendEventWithName:name body:body];
     } else {
-        NSDictionary *dictionary = @{
-            @"name": name,
-            @"data": body
-        };
+        NSDictionary *dictionary = [NSDictionary dictionaryWithObjectsAndKeys:
+            name, @"name",
+            body, @"data",
+            nil
+        ];
```

Comment:
`body` can be null for certain events, such as `RNCallKeepDidActivateAudioSession` which will fire before the bridge is up if answering a call really quickly!